### DISCUSSION TITLE
When py return more than one value i cannot parse it in c# code

### DISCUSSION THREAD

**Discussion by daxter44** [pythonnet/pythonnet#1995] [Q&A]
### Environment

-   Pythonnet version: 3.0.0
-   Python version: 3.8
-   Operating System: Win 11
-   .NET Runtime: 6.0

### Details

-   Describe what you were trying to get done.

    From C# env i run Python function doSomething() which return 4 int in that way : 
    return num1, num2, num3, num4 
    
    I dont know how to fetch this return from c# perspective, i tried assign results to List<int>,   Tuple<int, int, int, int> but always i got error, and i can use it only as PyObject but then i dont have access to this integers values... 
    How can i use data from this return in c# ?

**Comment by anovik** [comment]
I had the similar problem trying to use python function returning multiple values 

`num1, num2, num3, num4 = doSomething() `

The following worked for me:

`dynamic result = doSomething() `

Then you can use result[0] as num1, result[1] as num2, etc. So all these multiple values are packed into PyObject and you can get them by indexing.

Hope it helps.
